apps/i18n/common/en_us.json

```diff
@@ -523,6 +523,12 @@
   "findLocalClassButton": "Find a class",
   "finish": "Finish",
   "forTeachersOnly": "For Teachers Only",
+  "gdprDialogHeader": "Do you agree to using a web site based in the United States?",
+  "gdprDialogDetails": "Data from your use of this site may be sent to and stored or processed in the United States.",
```

Comment:
nit: we should also be consistent about the use of "website" or "site".

Reply:
We definitely need to use `website` instead of `web site` as we do in our privacy policy. I'm not too concerned about shortening to `site` since we've done that in the privacy policy too. But don't feel strongly about it. We can update all `site`s to `website` too.

Reply:
BTW, Erin I don't think you need to fix in this PR. I already fixed in mine https://github.com/code-dot-org/code-dot-org/pull/22541. I'll update mine though, since you're moving to React, so that it makes the adjustments to the apps strings only.